cequel.gemspec

```
@@ -29,7 +29,8 @@ DESC
 
   s.add_runtime_dependency 'activemodel', '>= 4.0'
   s.add_runtime_dependency 'bigdecimal', '~> 3.0'
-  s.add_runtime_dependency 'cassandra-driver', '~> 3.0'
+  # Require the ART19 patched version of cassandra-driver
+  s.add_runtime_dependency 'cassandra-driver', '3.2.5.1'
```

why no ~> here?

---

```suggestion
  s.add_runtime_dependency 'cassandra-driver', '~> 3.2.5.1'
```

Yeah this was me when we were debugging